statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -2607,7 +2607,9 @@ def plot_diagnostics(self, variable=0, lags=10, fig=None, figsize=None):
         # elements
         resid_nonmissing = resid[~(np.isnan(resid))]
         ax = fig.add_subplot(222)
-        ax.hist(resid_nonmissing, normed=True, label='Hist')
+        # temporarily disable Deprecation warning, normed -> density
```

Comment:
maybe add a TODO note specifying the conditions under which "temporarily" runs out?